pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -69,7 +69,11 @@ type storageClientV1 struct {
 
 // NewStorageClientV1 returns a new v1 StorageClient.
 func NewStorageClientV1(ctx context.Context, cfg Config, schemaCfg chunk.SchemaConfig) (chunk.IndexClient, error) {
-	opts := toOptions(cfg.GRPCClientConfig.DialOption(bigtableInstrumentation()))
+	dialOpts, err := cfg.GRPCClientConfig.DialOption(bigtableInstrumentation())
```

Comment:
Do we need this for Bigtable? Google seems to have their own way of doing this: https://godoc.org/google.golang.org/api/option#WithCredentials. It's based around GCP service accounts. I think we should avoid adding this for GCP services for now.